python/lsst/ts/criopy/m1m3ts/egw_pump_errors.py

```diff
@@ -0,0 +1,462 @@
+# This file is part of ts_xml.
+#
+# Developed for the LSST Telescope & Site.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["EGWPumpErrors"]
+
+from dataclasses import dataclass
+
+
+@dataclass
+class PumpError:
+    fault: str
+    fault_type: int
+    description: str
+    action: list[str]
+
+
+EGWPumpErrors = {
+    0: PumpError("No Fault", 0, "No fault present.", []),
+    2: PumpError(
```

Comment:
You can consider to use the enum instead of the magic number here. In addition, if these numbers come from the manual, maybe put the reference of manual as well?